wave_vision/tests/matcher_tests/brute_force_tests.cpp

```diff
@@ -327,4 +347,46 @@ TEST(BFTests, DISABLED_LMEDS) {
 
     cv::waitKey(0);
 }
+
+TEST(BFTests, DISABLED_SequenceMatches) {
```

Comment:
Yeah, in my other classes I only have 1 skipped test or so, but this one is just so image dependent. @leokoppel do you have any tips for testing this stuff a bit better? While trying to write tests I was struggling a bit with ensuring everything is tested properly. It's a bit tougher when visual results are pretty key for confirming operation.,